lib/node/Creator.js

```diff
@@ -71,6 +71,13 @@ const NodeCreator = {
   nthRoot (radicandNode, rootNode) {
     const symbol = NodeCreator.symbol('nthRoot');
     return new math.expression.node.FunctionNode(symbol, [radicandNode, rootNode]);
+  },
+  powerTerm (base, exponent) {
```

Comment:
Sadly, no. It is still needed in multiplyLikeTerms... hard to create the args for the node else.